.gitignore

```diff
@@ -2,9 +2,18 @@
 *~
 .*.swp
 *.pyc
+*.pyo
 *.retry
-host_vars/*
 docs/build
-hosts
 workspace
-files/*private*
+
+!__exekutir__(/.*)?/inventory/hosts
+
+jobs/.*/.*
+!jobs/.*/.empty
+
+__executir__/host_vars
```

Comment:
hmmmm, that's a good point about the spellings.  It use to trip me up until seeing/typing them a billion times.  I like how they "pop" because some of the plays are really tricky about the order or which hosts things happen on.  So, having them stand out in that regard isn't a bad thing.

I'm a bit loath to use "slave", because of the possible negative connotations and maybe that it's unimportant or to be disregarded.  The other problem is "commander" is awful similar to the name of an Ansible module, which could cause confusion.  Worse, if you accidentally slip a space into "executor" in just the wrong spot,  will most certainly cause a bad smell to emanate from your shell output 😀 

Though I'm not at all against changing the names to something else.  Do you have any better ideas? 

What about if we could "underline" them instead.  Like "__commander__" and "__executor__"  then they could match the directory names also.  I'm just not sure if ansible would accept that since you can't have underscores in DNS names.